ocaml/xapi-cli-server/cli_operations.ml

```diff
@@ -5349,28 +5348,8 @@ let with_license_server_changes printer rpc session_id params hosts f =
             ~value:license_server
         )
         current_license_servers ;
-      let alerts =
-        Client.Message.get_since ~rpc ~session_id
-          ~since:(Date.of_unix_time (now -. 1.))
-      in
-      let print_if_checkout_error (ref, msg) =
-        if
-          false
-          || msg.API.message_name = fst Api_messages.v6_rejected
-          || msg.API.message_name = fst Api_messages.v6_comm_error
-          || msg.API.message_name
-             = fst Api_messages.v6_license_server_version_obsolete
```

Comment:
If I understood this right, xe now doesn't need to know about these messages, but they still need to be exposed so clients can still react to them, is this correct?